.github/templates/dag_l1/initial_validator/action.yml

```diff
@@ -25,7 +25,7 @@ runs:
     - name: Generate DAG L1 JAR
       shell: bash
       run: |
-        sbt dagL1/assembly
+        sbt -Dversion.override=99.99.99 dagL1/assembly
```

Comment:
You can remove the 
-Dversion.override=99.99.99
it's unnecessary here

Reply:
:+1: